pkg/devfile/adapters/common/utils.go

```diff
@@ -191,3 +193,14 @@ func IsRestartRequired(command common.DevfileCommand) bool {
 
 	return restart
 }
+
+// GetCommandsMap returns a mapping of all of devfile command names to their corresponding DevfileCommand struct
+// Allowing us to easily retrieve the DevfileCommand of any command listed in a composite command
+func GetCommandsMap(commands []common.DevfileCommand) map[string]common.DevfileCommand {
```

Comment:
probably a unit test for this func since its exported

Reply:
Done